Update tea notebook with rotator plots

b1quint opened the pull request.

b1quint committed:
Update tea notebook with rotator plots

boutigny commented on notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:
```diff
@@ -109,9 +109,9 @@
     "# start = Time(\"2024-03-10T18:00:00Z\", scale=\"utc\", format=\"isot\")\n",
```
<p>That's a detail, but for me it takes 4.5 min</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/#comment-1684967907'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

b1quint replied:
I updated the message to warn about this.

boutigny commented on notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:
```diff
@@ -109,9 +109,9 @@
     "# start = Time(\"2024-03-10T18:00:00Z\", scale=\"utc\", format=\"isot\")\n",
```
<pre class="ql-syntax" spellcheck="false">Line #6.        long_name="SST M2 Surrogate",
</pre><p>Trivial, but what is the meaning of SST ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/#comment-1684967908'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

b1quint replied:
I added a comment to answer this question. SST stands for Simonyi Survey Telescope.

boutigny commented:
On top of the above very minor comments, I would suggest to have the same scales for rotator on and rotator off plots.
I also see that the plot corresponding to "BLOCK-197 - Hexapod Analysis" have no axis and no titles

boutigny reviewed (approved):
All comments are minor ones.

b1quint committed:
Updated notebook to address comments in the PR#140
- Add comments to identify what is SST
- Update hexapod analysis plots with titles and labels
- Update warning message

b1quint commented:
@boutigny I merged the PR you reviewed. I addressed some of the comments. The only comment I did not address was about the scales when comparing rotator on/off. That would be way too complicated.
